### PR TITLE
Create training.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Created training materials page as it was missing from the handbook.

I've tested all the links and they're all still working. 

@wiersgallak, @jwilander, @DSchalla, @jasonblais: This is from the old handbook and if you have any additional resources to add, please do. We do have a "professional development" page in this section of the handbook but as it is aimed at a smaller group of roles, I think this should be a standalone page. 0/5 on that.

Created to fix: https://github.com/mattermost/mattermost-handbook/pull/1088

### CODE REVIEW

File: operations/research-and-development/training.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+- [Framework for Improving Critical Infrastructure Cybersecurity. National Institute of Standards and Technology](https://templatelab.com/cybersecurity-framework) - Standards for internal Mattermost security processes and safeguards.
+- [Computer Security in the Real World. Butler Lampson](https://research.microsoft.com/en-us/um/people/blampson/69-SecurityRealIEEE/69-SecurityRealIEEE.pdf) - Fundamental challenges with system security.
+- [Course notes from CS513: System Security (Cornell University). Fred B. Schneider](https://www.cs.cornell.edu/courses/cs513/2007fa/02.outline.html) - Well written introduction to system security from one of the leaders in the field.
+
+## Additional training resources
+
+Recommended training materials are recommended by role at three different levels of priority:
+
+- P1 - Required - Complete within 30 days of starting in role.
```
Comment (jwilander) [inline operations/research-and-development/training.md]: I don't think we've ever enforced that new engineers read certain books within a timeframe of starting. Maybe we should update the language to make it clear that it's a recommendation and not required.

**Comment by jasonblais** [comment]
Thank you @justinegeffen, appreciate the help here. Do you know when the old handbook page was last updated? 

I wonder if this page is obsolete, as I haven't used these materials for PM materials at least

**Comment by justinegeffen** [comment]
@jasonblais, I think that page was updated before my time, so more than three years ago. With that in mind, maybe this *would* work better in the professional development page as a "nice to do but not mandatory"?

**Comment by jwilander** [comment]
>With that in mind, maybe this would work better in the professional development page as a "nice to do but not mandatory"?

+1

**Comment by jasonblais** [comment]
+1 for treating it as professional development :+1:

**Review by jwilander — APPROVED** [review]

**Pull request merged**